src/vue/backend/src/views/guided-setup/Agents.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <GuidedSetupContentCard
+        :cardTitle="cardTitle"
+        :steps="guidedSetupSteps"
+        :relatedLinks="links"
+        @back-step="goBackStep"
+        @skip-step="goSkipStep"
+        @continue="goContinue">
+        <template v-slot:description>
+            <p><strong>This is optional.</strong> A sentence or two about why you should install IMPress Listings to your WordPress site. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </template>
+        <template v-slot:controls>
+            <p>Activating IMPress Agents will:</p>
+            <idx-list customClass="list-featured">
+                <idx-list-item>Add Feature 1</idx-list-item>
+                <idx-list-item>Enable Feature 2</idx-list-item>
+                <idx-list-item>Import Feature 3</idx-list-item>
+                <idx-list-item>Automate Feature 4</idx-list-item>
+            </idx-list>
+            <idx-block className="form-content">
+                <idx-form-group>
+                    <idx-block className="form-content__toggle">
+                        {{ activateLabel }}
+                        <idx-toggle-slider
+                            uncheckedState="No"
+                            checkedState="Yes"
+                            @toggle="formUpdate({ key: 'enabled', value: !localStateValues.enabled })"
+                            :active="localStateValues.enabled"
+                            :label="activateLabel"
+                        ></idx-toggle-slider>
+                    </idx-block>
+                </idx-form-group>
+            </idx-block>
+        </template>
+    </GuidedSetupContentCard>
+</template>
+
+<script>
+import { mapState, mapActions } from 'vuex'
+import guidedSetupMixin from '@/mixins/guidedSetup'
+import pageGuard from '@/mixins/pageGuard'
+import GuidedSetupContentCard from '@/templates/GuidedSetupContentCard.vue'
+export default {
+    mixins: [
+        guidedSetupMixin,
+        pageGuard
+    ],
+    components: {
+        GuidedSetupContentCard
+    },
+    computed: {
+        ...mapState({
+            enabled: state => state.agentSettings.enabled,
```

Comment:
Are you using `enabled` here?

Reply:
Yeah, it's the activate plugin slider, and it's one of the store settings.

Reply:
The activate slider plugin is using `localStateValues`, not the state directly